h/models/auth_ticket.py

```diff
@@ -48,3 +49,6 @@ def generate_ticket_id() -> str:
         generate ids for most cases.
         """
         return urlsafe_b64encode(urandom(32)).rstrip(b"=").decode("ascii")
+
+    def __repr__(self):
+        return helpers.repr_(self, ["user_id", "expires"])
```

Comment:
This is the one case where I didn't include the object's ID in the repr: `AuthTicket.id` is both the primary (and only) key and also is the actual secret part of the authticket! So it's not safe to log that. This repr will at least make logging authtickets look friendlier and would probably be enough information to recover an authticket from a DB backup as well. Authtickets don't seem like a thing we'd likely really need to recover anyway